content/en/monitors/guide/monitor_best_practices.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Monitor Best Practices
+further_reading:
+- link: "/monitors/"
+  tag: "Documentation"
+  text: "Learn more about monitors"
+- link: "/monitors/configuration/"
+  tag: "Documentation"
+  text: "Monitor Configuration"
+---
+
+## Overview
+
+A well-designed monitor is essential for understanding the performance, health, and stability of your infrastructure or application. It provides accurate, actionable data to detect issues early, optimize performance, and maintain smooth operations.
+
+This guide walks you through the key aspects of building an effective monitor: 
+
+- Create an intuitive monitor  
+- Set proper thresholds  
+- Establish actionable responses. 
+
+By following these principles, you'll create a monitor that delivers meaningful insights and supports proactive decision-making.
+
+## Write a clear title
+
+The [Manage Monitor][1] page displays the monitor's title, monitor’s tags and team, and other fields that you can find [**here**][2]. Having a  clear, descriptive title is important because it immediately communicates the monitor's purpose and what it tracks. A well-defined title helps users quickly identify the monitor's focus. A clear title also ensures that all team members, regardless of role and persona, can understand the monitor's function at a glance. Please keep in mind that the monitor’s title and the notification subject are the same, so we suggest avoiding having too many variables in it.
+
+## Use multi-alert
+
+To avoid redundant monitors (such as multiple similar monitors with just 1 scope changing), Datadog recommends using multi-alert monitors. When you use a "group by" in your query, multi-alerts automatically apply the alert to each source based on your group parameters. The best way to achieve this is with a monitor grouped by “host”, so that you’ll have only 1 monitor that will trigger a distinct alert for each host every time it crosses the threshold.
```

Comment:
```suggestion
To avoid redundant monitors (such as multiple similar monitors with just 1 scope changing), Datadog recommends using multi-alert monitors. When you use a "group by" in your query, multi-alerts automatically apply the alert to each source based on your group parameters. The best way to achieve this is with a monitor grouped by `host`, so that you only have one monitor that triggers a distinct alert for each host every time it crosses the threshold.
```